website/package.json

```diff
@@ -39,7 +39,6 @@
   },
   "scripts": {
     "postinstall": "node public/getPosts.js",
-    "getContributors": "node public/getContributors.js",
```

Comment:
Please update the action file as well, if we are removing the script from website

Reply:
workflow yml is not present -> https://github.com/openebs/website/blob/revamp-3.0/.github/workflows/sanity-check.yml

https://github.com/openebs/website/blob/revamp-3.0/.github/workflows/eslint.yml

Reply:
No its in default branch

Reply:
done!